samtranslator/model/eventsources/push.py

```diff
@@ -743,6 +745,24 @@ def to_cloudformation(self, **kwargs):  # type: ignore[no-untyped-def]
         if explicit_api.get("__MANAGE_SWAGGER") or explicit_api.get("MergeDefinitions"):
             self._add_swagger_integration(explicit_api, api_id, function, intrinsics_resolver)  # type: ignore[no-untyped-call]
 
+        swagger_body = explicit_api.get("DefinitionBody")
+
+        if swagger_body and self.Auth and self.Auth.get("TestAuthOverride") is True:
```

Comment:
I thought the name `TestAuthOverride` is still pending UX decision based on the doc I saw yesterday. Has that decision been made?

Reply:
Not yet, I've signed up for tomorrows UX office hour and will push a change to update the property name once that decision is made. The current PR is open just for general review and will not be merged until the property name is updated.